style.css

```diff
@@ -235,6 +273,52 @@ input {
   margin-left: 28px;
 }
 
+@media screen and (min-width: 768px) {
+  .container-2 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .image-header {
```

Comment:
- [ ] Kindly use the correct background image (desktop one) for desktop view, this will make your design closer to the Figma one

|Your design|Figma design|
|-|-|
|![Screenshot_20220611_093047](https://user-images.githubusercontent.com/25227022/173180339-dd602432-5633-4317-b0b7-d7c1d550160f.png)|![Screenshot_20220611_093215](https://user-images.githubusercontent.com/25227022/173180345-12b3f9fb-aa52-4ebe-89f0-381ef6eed498.png)|